### PR TITLE
Video creative fix

### PR DESCRIPTION
**Pull request by jitendra-xoriant**
added support to GET https://<<Host>>/video/v2.0/creatives/12345

### CODE REVIEW

File: src/main/java/com/mediamath/terminalone/QueryCriteria.java [hunk 1]
```diff
@@ -26,6 +26,10 @@
 import com.mediamath.terminalone.utils.QueryParamValues;
 
 public class QueryCriteria {
+	
+	public enum CreativeType {
+	    video, normal;
+	}
```
Comment (karthik-mediamath) [inline src/main/java/com/mediamath/terminalone/QueryCriteria.java]: we also have a dynamic creative, html5 (more like GIFs). Will normal cover that as well?

File: src/main/java/com/mediamath/terminalone/models/VideoCreative.java [hunk 2]
```diff
@@ -1,177 +1,199 @@
...
+  private boolean mobileEncoding;
+  private long concept;
+  private String clickthroughUrl;
+  private boolean active;
+  private ArrayList<eventPixelsEnum> eventPixels = new ArrayList();
+  private long creativeId;
+
+  public enum eventPixelsEnum {
+
+    ImpSkippable("imp:skippable"), Complete("complete"), CreativeClick(
```
Comment (karthik-mediamath) [inline src/main/java/com/mediamath/terminalone/models/VideoCreative.java]: Fix indentation here

File: src/main/java/com/mediamath/terminalone/service/T1Service.java [hunk 3]
```diff
@@ -106,6 +106,10 @@ public String constructReportingUrl(StringBuilder path) {
 	public String constructOauthUrl(StringBuilder path) {
 		return oauthURL + "/" + path.toString();
 	}
+	public String constructVideoCreativeUrl(StringBuilder path) {
```
Comment (karthik-mediamath) [inline src/main/java/com/mediamath/terminalone/service/T1Service.java]: we had video creatives support previously. How different is this construct url from previous?

File: src/main/java/com/mediamath/terminalone/QueryCriteria.java (hunk 1 above)
Comment (jitendra-xoriant) [inline src/main/java/com/mediamath/terminalone/QueryCriteria.java]: yes, as URL for all except video is same.

File: src/main/java/com/mediamath/terminalone/service/T1Service.java (hunk 3 above)
Comment (jitendra-xoriant) [inline src/main/java/com/mediamath/terminalone/service/T1Service.java]: For previous Supported POST API, we do not use any method from T1Service, so I have to create one method in T1Service, which can be accessed in TerminalOne class.

File: src/main/java/com/mediamath/terminalone/models/VideoCreative.java (hunk 2 above)
Comment (jitendra-xoriant) [inline src/main/java/com/mediamath/terminalone/models/VideoCreative.java]: done

File: src/main/java/com/mediamath/terminalone/service/T1Service.java (hunk 3 above)
Comment (karthik-mediamath) [inline src/main/java/com/mediamath/terminalone/service/T1Service.java]: So, previously this url was constructed in a different place? just trying to understand

Comment (jitendra-xoriant) [inline src/main/java/com/mediamath/terminalone/service/T1Service.java]: yes, in PostService class

**Pull request merged**